src/types.ts

```diff
@@ -0,0 +1,31 @@
+import type supportedLanguages from './supportedLanguages.js';
+import type { ClientId, TargetId } from '@readme/httpsnippet/targets';
+import type { AuthForHAR, DataForHAR } from '@readme/oas-to-har/lib/types';
+
+export type { AuthForHAR, DataForHAR };
```

Comment:
Can't export them like that because they become local types and clash with the one we're importing:

![Screen Shot 2023-10-20 at 9 21 47 AM](https://github.com/readmeio/oas-to-snippet/assets/33762/ff0d9b8b-76ed-4762-9fc3-58271d13fdae)

We don't seem to be using either of these two types in the main app though so I'm going to stop re-exporting them.